composer.json

```diff
@@ -15,14 +15,13 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "~4.2|~5.0"
+        "illuminate/support": "~5.0"
```

Comment:
You should really add contracts as a dependency, even though it's currently required by support.